brilirs/src/interp.rs

```diff
@@ -516,6 +518,32 @@ fn execute_value_op<T: std::io::Write>(
       let res = Value::Pointer(arg0.add(arg1));
       state.env.set(dest, res);
     }
+    Float2Bits => {
+      let input = get_arg::<&Value>(&state.env, 0, args);
+
+      let (Value::Float(float), Type::Int) = (input, op_type) else {
+        panic!("Invalid bitcast");
+      };
+
+      // https://users.rust-lang.org/t/i64-u64-mapping-revisited/109315
+      // the to and from native endian stuff is a nop
+      // if link dies try web archive
+      let uint = float.to_bits();
+      let int = i64::from_ne_bytes(uint.to_ne_bytes());
```

Comment:
Nit: Just to check my understanding here, is it necessary to produce an intermediate u64 here as opposed to using `f64::to_ne_bytes`?